README.md

```diff
@@ -20,7 +20,7 @@ Here is a list of supported languages:
 |:---:|:---:|
 |1 |Bash (4.4)|
 |2 |Bash (4.0)|
-|3 |Basic (fbc 1.05.0)|
+|3 |Basic (fbc 1.07.1) |
```

Comment:
Same as the previous comment. This cannot get into new release because it will break Judge0 API major version. This is essentially API change.

Reply:
Well, it's a minor change isn't it? The id and the major version of the language are the same. We'll just have to update the seeds.rb file.